loginas/utils.py

```diff
@@ -0,0 +1,63 @@
+import logging
+
+from django.conf import settings as django_settings
+from django.contrib.auth.models import User
+from django.contrib.auth import load_backend, login, logout
+from django.contrib import messages
+from django.core.signing import TimestampSigner, SignatureExpired
+from datetime import timedelta
+
+from . import settings
```

Comment:
It does have something to do with django_settings since it's where the django conf stack is imported.. actually IMO the nicest thing for a reusable app would be to only have one settings module where `django.conf.settings` is used. That way, everything that's configurable is accessed through the same module interface.

I changed it to `la_settings` anyways as it's no big deal :D Don't get me wrong, I like to discuss best practice, but no holy cows here.... moo-ving on :)